packages/dashboard/src/app-config.ts

```diff
@@ -7,6 +7,7 @@ export interface AppConfig {
   authenticator: Authenticator;
   trajServerUrl: string;
   rmfServerUrl: string;
+  isKeycloak: boolean;
```

Comment:
We can just use the `REACT_APP_AUTH_PROVIDER` env, suppose we have N auth providers, it wouldn't be nice to have `isProvider1`, `isProvider2`...